kotest-framework/kotest-framework-engine/src/commonMain/kotlin/io/kotest/engine/spec/Materializer.kt

```diff
@@ -68,9 +68,18 @@ class Materializer(private val configuration: ProjectConfiguration) {
     * Materializes a [NestedTest] into a runtime [TestCase] by attaching it to the given parent.
     */
    fun materialize(nested: NestedTest, parent: TestCase): TestCase {
+
+      // Note: intellij has a bug, where if a child test has a name that starts with the parent test name,
+      // then it will remove the common prefix from the child, to workaround this, we will add a dash at the
+      // start of the nested test to make the child nest have a different prefix.
+      // Also note: This only affects non-MPP tests, as MPP tests have the platform name added
```

Comment:
Can you link/ref the bug from their tracker so developers can check it later to see if it's been actioned?

Reply:
It's more an undocumented feature I guess